osmosis-1/osmosis-1.assetlist.json

```diff
@@ -1214,6 +1214,37 @@
       },
       "coingecko_id": "neta"
     },
+    {
+      "description": "The native token cw20 for Marble DAO on Juno Chain",
+      "type_asset": "cw20",
+      "address": "juno1g2g7ucurum66d42g8k5twk34yegdq8c82858gz0tq2fc75zy7khssgnhjl",
+      "denom_units": [
+        {
+          "denom": "ibc/F6B691D5F7126579DDC87357B09D653B47FDCE0A3383FF33C8D8B544FE29A8A6",
+          "exponent": 0,
+          "aliases": [
+            "umarble"
+          ]
+        },
+        {
+          "denom": "marble",
+          "exponent": 3
+        }
+      ],
+      "base": "ibc/F6B691D5F7126579DDC87357B09D653B47FDCE0A3383FF33C8D8B544FE29A8A6",
+      "name": "Marble",
+      "display": "marble",
+      "symbol": "MARBLE",
+      "ibc": {
+        "source_channel": "channel-47",
+        "dst_channel": "channel-169",
+        "source_denom": "umarble"
+      },
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/marble-logo.png"
```

Comment:
`        "png": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/marble.png"
`
Also can you rename image -> marble.png